service_mesh/v2x/ossm-reference-jaeger.adoc

```diff
@@ -8,6 +8,11 @@ toc::[]
 
 When the {SMProductShortName} Operator deploys the `ServiceMeshControlPlane` resource, it can also create the resources for distributed tracing. {SMProductShortName} uses Jaeger for distributed tracing.
 
+[IMPORTANT]
+====
+Jaeger does not use FIPS validated cryptographic modules.
```

Comment:
I'd make sure capitalization of `validated` is consistent with above uses of `FIPS Validated or Modules in Process cryptographic...`